### ISSUE TITLE
Querying punctuation with yql

### ISSUE THREAD

**Issue by jeffkayne** [vespa-engine/vespa#12508]
As mentioned on gitter, I am trying to query data (with a yql), to find all documents that match the string "&Free".
The exact query I'm executing is:
"SELECT text_raw FROM sources post WHERE text_raw CONTAINS '&free';"
This query only filters documents that contain the string "free", not "&free".

The specific use-case for this is that I am looking for brand names within documents.

Vespa version: 7.165.5

**Comment by bratseth** [comment]
Is the use case here that "&free" is a proper name that you need to match?

**Comment by jeffkayne** [comment]
Yes, correct

**Comment by jobergum** [comment]
Please see match modes https://docs.vespa.ai/documentation/reference/search-definitions-reference.html#match. Default for string fields is text which implies tokenization where punctation characters are removed.

**Comment by bratseth** [comment]
I think this is to make "special tokens" work independently of the linguistics plugin then. I'll try to find some time somewhere to do that.

**Comment by jeffkayne** [comment]
Yes that would be great if we could use an escape, for example "\x##" to get an exact match of the string. Cheers!

**Comment by bratseth** [comment]
That's no problem but doesn't solve the problem because the engine need to decide at wqrite time what tokens to index.

**Comment by baldersheim** [comment]
**soon** timed out
